bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -244,6 +254,12 @@ case object SimbadAdapter extends CatalogAdapter {
     }
     \/-((band.flatten |@| MagnitudeSystem.fromString(p._2))((b, s) => (b, s)))
   }
+
+  def containsExceptions(xml: Node): Boolean = {
+    // The only case known is with java.lang.NullPointerException but let's make the check
+    // more general.
+    (xml \\ "INFO" \ "@value").text.matches("java\\..*Exception")
```

Comment:
I suppose there could there could be `*Error`s or exceptions particular to some library other than the standard library?  I guess detecting this is tricky though.